nntrainer/utils/util_simd.cpp

```diff
@@ -40,6 +40,39 @@ void swish(const unsigned int N, float *X, float *Y, float *Z) {
 #endif
 }
 
+float max(const unsigned int N, float *X) {
+#ifdef USE_NEON
+  return nntrainer::neon::max(N, X);
+#else
```

Comment:
Use STL properly

```C
std::vector<float> v (X, X+N);
return *std::max_element(v.begin(), v.end());
```

And if you compile it properly, you may get x64/SIMD (AVX/SSE) for free:

https://stackoverflow.com/questions/59373900/why-is-there-no-simd-functionality-in-the-c-standard-library